go/service/wot.go

```diff
@@ -0,0 +1,54 @@
+package service
+
+import (
+	"github.com/keybase/client/go/engine"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/go-framed-msgpack-rpc/rpc"
+	"golang.org/x/net/context"
+)
+
+type WebOfTrustHandler struct {
+	*BaseHandler
+	libkb.Contextified
+}
+
+func NewWebOfTrustHandler(xp rpc.Transporter, g *libkb.GlobalContext) *WebOfTrustHandler {
+	return &WebOfTrustHandler{
+		BaseHandler:  NewBaseHandler(g, xp),
+		Contextified: libkb.NewContextified(g),
+	}
+}
+
+func (h *WebOfTrustHandler) WotVouch(ctx context.Context, arg keybase1.WotVouchArg) error {
```

Comment:
if this command is called "vouch" rather than "attest", I'd like to go back and change the name on the server. should i do that now?

Reply:
We should discuss with @malgorithms in chat:  vouch was his idea.  I like vouch.